### ISSUE TITLE
Spotify Web API auth flow - illegal scope, only after quota extension granted

### ISSUE THREAD

**Issue by jakecollins1** [spotify/web-api-examples#114]
My spotify web api project was working with my approved spotify account with the `user-top-read` scope.  As soon as my extension request was granted, my spotify auth flow stopped working, with the spotify/authorize page not redirecting me and instead displaying the message "Illegal scope"

**Comment by ImJonathanZ** [comment]
I have been getting the same error as well. Only started using this project today though. Perhaps your problem is not with the quota extension and maybe another issue within the project?
